recipe-server/normandy/recipes/api/views.py

```diff
@@ -62,7 +62,7 @@ def retrieve(self, request, name, impl_hash):
 
 
 class RecipeFilters(django_filters.FilterSet):
-    enabled = CaseInsensitiveBooleanFilter(name='enabled', lookup_type='eq')
+    enabled = CaseInsensitiveBooleanFilter(name='enabled', lookup_expr='eq')
```

Comment:
Just curious, why update django-filters and change this stuff in this PR? I'm fine with it though.